app/models/vocabulary.rb

```diff
@@ -1,6 +1,15 @@
 class Vocabulary < ApplicationRecord
-  validates :string_key, presence: true, uniqueness: true
+  has_many :term, dependent: :destroy
```

Comment:
```suggestion
  has_many :terms, dependent: :destroy
```
I think that this should work with the plural form (and that's what I'm used to from the Rails docs).

Reply:
okay, will change